tests/vmi_multus_test.go

```diff
@@ -1052,3 +1058,16 @@ func checkSriovEnabled(virtClient kubecli.KubevirtClient, sriovResourceName stri
 	}
 	return false
 }
+
+func findSRIOVInterface(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, networkName string) string {
+	var err error
+	vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Comment:
can you add a specific expect description so that if it ever fails, you could find where it happened easier?

Reply:
i think that in this specific case the red dump would say exactly the vm wasnt found , isnt i t?

Reply:
if the test fails here then we know of course that the vmi.get failed from the error, but where not where exactly. I mean, we run many `virtClient.VirtualMachineInstance(vmi.Namespace).Get` along the tests, how would be able to pin point which one of them was it? that's what I meant by "specific description".. 
it's not always preferable, but can help future debugging IMO.

Reply:
it will show you the exact line in the test that invoked this function, because there is ExpectWithOffset

Reply:
in that case, I withdraw any objection